src/lemlib/odom/differentialArc.cpp

```diff
@@ -138,33 +138,36 @@ void DifferentialArc::update() {
         infoSink()->error("Odom calculation failure! Not enough sensors to calculate heading");
         return;
     }
-    const float deltaTheta = theta - pose.theta;
-    const float avgTheta = pose.theta + deltaTheta / 2;
+    const Angle deltaTheta = theta - pose.theta;
+    const Angle avgTheta = pose.theta + deltaTheta / 2;
 
     // calculate local change in position
-    Pose local(0, 0, deltaTheta);
+    Pose local(0_in, 0_in, deltaTheta);
     // set sinDTheta2 to 1 if deltaTheta is 0. Simplifies local position calculations.
-    const float sinDTheta2 = (deltaTheta == 0) ? 1 : 2 * std::sin(deltaTheta / 2);
+    const float sinDTheta2 = (deltaTheta == 0_deg) ? 1 : 2 * units::sin(deltaTheta / 2).raw();
 
     // calculate local y position
     for (auto& tracker : horizontals) {
         // prevent divide by 0
-        const float radius = (deltaTheta == 0) ? tracker.getDistanceDelta()
-                                               : tracker.getDistanceDelta() / deltaTheta + tracker.getOffset();
+        const Length radius = (deltaTheta == 0_deg) ? tracker.getDistanceDelta()
+                                                    : tracker.getDistanceDelta() / deltaTheta.convert(rad) +
+                                                          tracker.getOffset(); // todo test
         local.y += sinDTheta2 * radius / horizontals.size();
     }
 
     // calculate local x position
     if (verticals.size() > 0) { // use dedicated tracking wheels if we have any
         for (auto& tracker : verticals) {
-            const float radius = (deltaTheta == 0) ? tracker.getDistanceDelta()
-                                                   : tracker.getDistanceDelta() / deltaTheta + tracker.getOffset();
+            const Length radius = (deltaTheta == 0_deg)
```

Comment:
```suggestion
            const Length radius = (deltaTheta == 0_rad)
```
Change to radians for consistency